data_processors/pipeline/domain/workflow.py

```diff
@@ -353,6 +353,14 @@ def must_be_tumor(self):
             raise LabMetadataRuleError(f"Not tumor sample.")
         return self
 
+    def must_be_tumor_or_ntc(self):
+        """Sample Phenotype must be tumor"""
+        from data_portal.models.labmetadata import LabMetadataPhenotype
+        if self.this_metadata.phenotype.lower() == LabMetadataPhenotype.TUMOR.value.lower() or self.this_metadata.phenotype.lower() == LabMetadataPhenotype.N_CONTROL.value.lower():
```

Comment:
I think for consistency this should be `if ... then fail else return self`

The following would negate this logic:

```python
if not (
    self.this_metadata.phenotype.lower() == LabMetadataPhenotype.TUMOR.value.lower() or 
    self.this_metadata.phenotype.lower() == LabMetadataPhenotype.N_CONTROL.value.lower()
    ):
    raise LabMetadataRuleError(f"Not tumor sample or NTC sample.")
return self

```